content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -433,8 +433,12 @@ The following table below lists configuration variables that are available for b
 | `DD_TRACE_HEADER_TAGS`                              | Accepts a map of case-insensitive header keys to tag names and automatically applies matching header values as tags on root spans. Example: `CASE-insensitive-Header:my-tag-name,User-ID:userId`. Added in version 1.18.3.  |
 | `DD_TRACE_LOG_DIRECTORY`                            | Sets the directory for .NET Tracer logs.<br/><br/>Default: `%ProgramData%\Datadog .NET Tracer\logs\`      |
 | `DD_TRACE_LOG_PATH`                                 | Sets the path for the automatic instrumentation log file and determines the directory of all other .NET Tracer log files. Ignored if `DD_TRACE_LOG_DIRECTORY` is set.             |
+| `DD_TRACE_LOGGING_RATE`                             | Sets rate limiting for log messages. If set, unique log lines will be written once per `x` seconds. For example, to log a given message once per 60s, set to `60`. Setting to `0` disables log rate limiting. Disabled by default |
```

Comment:
In what version was this setting added?

Reply:
That's a slightly complicated question - it was originally added in `1.24.0`, enabled by default. We then disabled it during debug mode by default in `1.25.0`, and in `1.26.0` we disable it by default entirely. I'm guessing I should put `1.24.0+`, though the behaviour specifically describes `1.26.0`?

Reply:
I think `Added in version 1.24.0` is fine since that's the version where users can start using this, and setting it to a non-default value has the same behavior in all those versions. (right?)

Reply:
done in https://github.com/DataDog/documentation/pull/10221/commits/51e9f8bdf7ab18c1f41b3d6b64bb496b7755c008